src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int side;
+
+    public RightTriangle(int side, String color) {
+        this.side = side;
+        this.color = color;
+    }
+
+    public int getSide() {
+        return side;
+    }
+
+    public void setSide(int side) {
+        this.side = side;
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println(toString());
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: rightTriangle, area: "
+                + Math.sqrt(3) * side * side / 4
```

Comment:
```suggestion
                + getArea()
```
add interface AreaCalculator with method getArea
and Override this method in each of your figures